modules/storage-persistent-storage-pv.adoc

```diff
@@ -120,11 +120,11 @@ endif::[]
 
 [IMPORTANT]
 ====
-A volume's `AccessModes` are descriptors of the volume's capabilities. They
+A volume's access modes are descriptors of the volume's capabilities. They
```

Comment:
s/A volume's access modes are descriptors of the volume's capabilities./Volume access modes are descriptors of volume capabilities./